### PR TITLE
WINC-726: Add VM hardware version in Packer config

### PR DESCRIPTION
**Pull request by saifshaikh48**
This is in order for golden images built using our docs as a guide have
the proper VM hardware version in vSphere environments.

### CODE REVIEW

File: docs/vsphere_ci/README.md
Comment (saifshaikh48) [inline docs/vsphere_ci/README.md:17]: @jrvaldes we can continue [this conversation](https://github.com/openshift/windows-machine-config-operator/pull/892#discussion_r798783705) here

Comment (aravindhp) [reply docs/vsphere_ci/README.md:17]: My take has always been to use the version we have tested with. We don't know how the newer versions will work. What if there are breaking changes?

Comment (jrvaldes) [reply docs/vsphere_ci/README.md:17]: @saifshaikh48, given you [used](https://github.com/openshift/windows-machine-config-operator/pull/892#discussion_r798803541) Packer 1.6.6, I suggest removing the `+` .

File: docs/vsphere-golden-image.md
Comment (jrvaldes) [inline docs/vsphere-golden-image.md:15]: Remove `WINC-726`  from the commit message.

File: docs/vsphere-golden-image.md
Comment (jrvaldes) [inline docs/vsphere-golden-image.md:19]: Consider moving this change to another commit with a clear commit message. 
For example:  
```
[docs] Set hardware version in vSphere

...
```

Easier to review by docs Team.

Comment (jrvaldes) [inline docs/vsphere-golden-image.md:19]:
```suggestion
See [vSphere documentation](https://docs.vmware.com/en/VMware-vSphere/index.html) for installation guidelines. The supported [virtual hardware version](https://kb.vmware.com/s/article/1003746) is `15`.
```

Comment (saifshaikh48) [reply docs/vsphere-golden-image.md]: I don't see this as necessary

File: docs/vsphere_ci/README.md
Comment (saifshaikh48) [reply docs/vsphere_ci/README.md]: Makes sense, removed.

File: docs/vsphere-golden-image.md
Comment (jrvaldes) [reply docs/vsphere-golden-image.md]: Re-reading this, we don't need to add another maintenance point WRT the hardware version.

For client-facing docs we should NOT set any explicit version, just a reference to [VMware vSphere infrastructure requirements](https://docs.openshift.com/container-platform/4.9/installing/installing_vsphere/preparing-to-install-on-vsphere.html#installation-vsphere-infrastructure_preparing-to-install-on-vsphere) for OCP will suffice, it reads:

File: docs/vsphere-golden-image.md
Comment (sebsoto) [inline docs/vsphere-golden-image.md:19]: My info might be out of date, but I thought we were going to link to the [OpenShift docs](https://docs.openshift.com/container-platform/4.9/updating/updating-hardware-on-nodes-running-on-vsphere.html) for this information?

This should be OpenShift release specific and 4.9 supports 13+?

Comment (saifshaikh48) [reply docs/vsphere-golden-image.md:19]: The 4.9 docs have a note:
```
virtual hardware version 13 is now deprecated. These versions are still fully supported, but support will be removed in a future version of OpenShift Container Platform. Hardware version 15 is now the default
```
But yes, I agree, I'll change this point to OpenShift docs and not mention a any specific version here.

File: docs/vsphere-golden-image.md
Comment (jrvaldes) [reply docs/vsphere-golden-image.md]: See https://github.com/openshift/windows-machine-config-operator/pull/515/commits for references.

File: docs/vsphere-golden-image.md
Comment (sebsoto) [reply docs/vsphere-golden-image.md]: Wow thats confusing as it contradicts the first paragraph of that page :laughing:

**Comment by jrvaldes** [comment]
/lgtm

Thanks @saifshaikh48  for working on this.

**Comment by jrvaldes** [comment]
/hold 

Needs LGTM from Docs Team

**Review by codyhoag — APPROVED** [review]
LGTM

**Comment by saifshaikh48** [comment]
/cherry-pick release-4.9

furthest backport to 4.9 since 4.8 and below are only in maintenance support.

**Pull request merged**
